deployments/production/spark-master-controller.json

```diff
@@ -0,0 +1,86 @@
+{
+    "apiVersion": "extensions/v1beta1",
+    "kind": "Deployment",
+    "metadata": {
+        "generation": 3,
```

Comment:
@vellichor let's remove it. You're absolutely correct; it's cluster-side metadata.